docs/docs/config/kscan.md

```diff
@@ -0,0 +1,309 @@
+---
```

Comment:
Is it also worth refactoring `/docs/docs/features/debouncing.md` to refer to this page for configuration, similar to combos and underglow?

Reply:
I feel like debouncing.md is probably more useful as its own page, since it describes only debouncing and doesn't mention any of the other kscan config options that you only need when making a new board/shield.